src/core/ui/components/drop-down/inner.tsx

```diff
@@ -24,6 +24,7 @@ import { useEnvironment } from '../../context/environment';
 import { User } from '../../../user';
 import { EthLogo } from '../icon/ethereum-eth-logo';
 import { WalletConnectLogo } from '../icon/wallet-connect-logo';
+import { EtherscanLogo } from '../icon/etherscan-logo';
```

Comment:
Was this file checked in as part of another commit? I don't see it here

Reply:
it was it in the components/icon folder and didn't show up in git add. moved it to drop-down/icons